modules/is_master/manifests/params.pp

```diff
@@ -0,0 +1,71 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+class is_master::params {
+
+  $user = 'wso2carbon'
+  $user_group = 'wso2'
+  $product = 'wso2is'
+  $product_version = '5.7.0'
+  $service_name = 'wso2is'
+
+  # Define the templates
+  $start_script_template = 'bin/wso2server.sh'
+  $build_script_template = 'build.sh'
+
+  $template_list = [
+    'repository/conf/carbon.xml',
+    'repository/conf/user-mgt.xml',
+    # 'repository/conf/datasources/master-datasources.xml',
```

Comment:
Any special reason why these are commented?

Reply:
They were commented out in the original module for IS. I kept most of the params file as it is when configuration changes were moved over to is_master.

Reply:
Shall we uncomment  them? so the main config changes could be applied directly by changing the erb templates.

Reply:
+1

Reply:
https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b